examples/StandardFirmataWiFi/StandardFirmataWiFi.ino

```diff
@@ -309,6 +309,12 @@ void checkDigitalInputs(void)
 }
 
 // -----------------------------------------------------------------------------
+// function forward declarations
+void enableI2CPins();
```

Comment:
After reordering the functions in the sketch the forward definitions can be removed. But there is also the question of context, of keeping certain functions in a textual proximity or order to each other.